DomainContent/FaceToFace/StageManager/stageManager.js

```diff
@@ -63,8 +65,26 @@
     function onWebEventReceived(event) {
         if (typeof event === 'string') {
             findTargets();
+            print(event);
             event = JSON.parse(event);
             switch (event.type) {
+                case 'setWebOverlayUrl' : 
+                    webOverlayURL = event.url;
+                    break;
+                case 'spawnWebOverlay' :
+                    webOverlaySpawner = Entities.addEntity({
+                        type: 'Box',
+                        position: stageCenterPosition,
+                        visible: true,
```

Comment:
Is this meant to be visible: true?

Reply:
Just tried spawning it and you can see it from the stage.  Wasn't sure if it was intended